packages/grafana-data/src/utils/dataLinks.ts

```diff
@@ -57,11 +62,15 @@ export function mapInternalLinkToExplore(options: LinkToExploreOptions): LinkMod
           // Explore data links can be displayed not only in DataLinkButton but it can be used by the consumer in
           // other way, for example MenuItem. We want to provide the URL (for opening in the new tab as well as
           // the onClick to open the split view).
-          event.preventDefault();
+          if (event.preventDefault) {
```

Comment:
It looks like some types are incorrect if an object that is not an event can be passed here. Could you mention it in #75202 that it should be checked?

Reply:
Done!